packages/alfa-css/src/value/transform/scale.ts

```diff
@@ -64,10 +107,15 @@ export class Scale
       ...super.toJSON(),
       x: this._x.toJSON(),
       y: this._y.toJSON(),
+      z: this._z !== undefined ? this._z.toJSON() : null,
```

Comment:
**Note:** We can also get rid of `z` totally by doing a trick like
```
...(this._z !==  undefined ? { z: this._z.toJSON() } : {})
```
I'm never totally sure what is the best solution 😅 The trick makes for more compact JSON (and avoid the need to update a lot of code relying on it, so better backwards compatibility), but is also quite ugly to write and read/understand…